### ISSUE TITLE
Organize the PnP Core SDK Documentation alphabetically

### ISSUE THREAD

**Issue by MathijsVerbeeck** [pnp/pnpcore#855]
## Category

- [x] Feature request

## Describe the feature

Not really a feature request, but when currently navigating through the PnP Core Documentation (Using the SDK), the documentation is ordered quite 'randomly' I think. 

I agree that 'Overview' should always be listed as the first item, but the child items should, in my opinion, be ordered alphabetically.

If you don't agree, please don't hesitate to correct me, but it kinda _triggers_ me in a way that it's not ordered alphabetically.

Example:

![image](https://user-images.githubusercontent.com/9193870/169619453-55dae5b7-f730-46a4-8c11-a9fc8a52b5cc.png)

## Describe the solution you'd like

![image](https://user-images.githubusercontent.com/9193870/169619557-0e994190-047c-497b-9537-c2c66d7736d1.png)

## Additional context

N/A

**Comment by jansenbe** [comment]
Makes sense @MathijsVerbeeck , now that the documentation tree is growing spotting what you need is harder when not ordered. I'll make some updates, it will not be 100% alphabetical, but much closer :-)

**Comment by jansenbe** [comment]
done
